src/main/java/net/mcreator/minecraft/MCItem.java

```diff
@@ -80,6 +81,14 @@ protected void setIcon(ImageIcon icon) {
 			this.icon = icon;
 	}
 
+	public boolean isPOI() {
```

Comment:
Seeing this method is only used once, I think it had better be inlined, for example, like this:
`MCItem::isPOI`
<kbd>-></kbd>
`e -> e.getOther() instanceof Map<?, ?> m && "true".equals(m.get("poi").toString())`

Reply:
This may be used in other locations and I think JIT inlines this anyway currently

Reply:
Well, it just looks kinda too specific.